.babelrc

```diff
@@ -1,4 +1,13 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-object-assign", "transform-object-rest-spread"],
+  "presets": ["react", ["env", {
+    "targets": {
+      "browsers": ["last 2 versions", "iOS >= 10"]
+    },
+    "modules": false
```

Comment:
this stops babel from transpiling the import/export statements. This enables webpack to process those imports and do proper tree shaking